storage/typeprovider.go

```diff
@@ -8,11 +8,11 @@ import (
 type Provider interface {
 	Get(alias string) (string, error)
 	Exists(alias string) (bool, error)
-	Store(url, alias string) error
+	Store(url, alias string, overwrite string) error
```

Comment:
```suggestion
	Store(url, alias string, overwrite bool) error
```
This can/should be a bool. You're getting the value from the server as a string, which means at that layer of the program you'll need to address that. Here tho, you control what the interface requires and how it should take in parameters, so a bool is more appropriate